modules/adfs/lib/IdP/ADFS.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+use RobRichards\XMLSecLibs\XMLSecurityDSig;
+use RobRichards\XMLSecLibs\XMLSecurityKey;
```

Comment:
What change are these necessary for?

Reply:
They're to silence Scrutinizer complaining about missing namespaces.
We also do this in `modules/saml/lib/Message.php` and `modules/saml/lib/IdP/SAML2.php`, so it makes sense, I think?